tree/10_ncf_internals/configuration.cf

```diff
@@ -79,6 +79,19 @@ bundle agent configuration
       "pass2" expression => "pass1";
       "pass1" expression => "any";
 
+      "info"         expression => "inform_mode",
```

Comment:
@peckpeck @ncharles Now:

* Options defined by CFEngine will define the matching ncf verbosity class (but verbose in CFEngine = debug in ncf, and debug in CFEngine = trace in ncf)
* ncf classes are incrementally defined